dspace/config/clarin-dspace.cfg

```diff
@@ -172,5 +172,9 @@ google.analytics.cron = 0 0/1 * * * ?
 google.analytics.api-secret = EvBfmLs_QPaKei9XRibauw
 
 
+#Email to contact DSpace
+dspace.contact.email = marketa.trykarova@tul.cz
```

Comment:
I'm thinking if we could use `lr.help.mail` configuration property instead of creating a new one. `lr.help.mail` is exactly for the help desk.